enterprise/frontend/src/embedding-sdk/components/private/SdkUsageProblem/SdkUsageProblemBanner.tsx

```diff
@@ -27,27 +28,27 @@ export interface SdkUsageProblemBannerProps {
 // so they remain legible even when the theme is changed.
 const unthemedBrand = originalColors["brand"];
 const unthemedTextDark = originalColors["text-dark"];
-const unthemedTextMedium = originalColors["text-medium"];
 
 export const SdkUsageProblemBanner = ({
   problem,
 }: SdkUsageProblemBannerProps) => {
   const theme = useMantineTheme();
 
   const [expanded, setExpanded] = useState(false);
-  const [shown, { close: hideBanner }] = useDisclosure(true);
 
-  if (!problem || !shown) {
+  if (!problem) {
```

Comment:
We no longer allow dismissing the banner in the new design. We can only expand or collapse it.

<img src="https://github.com/user-attachments/assets/743a5211-37af-4336-a136-3491fa6733a3" width="300">